docs/reference/func_deploy.md

```diff
@@ -101,7 +101,7 @@ func deploy
 
 ```
       --build string[="true"]   Build the function. [auto|true|false]. [Env: $FUNC_BUILD] (default "auto")
-  -b, --builder string          builder to use when creating the underlying image. Currently supported builders are "pack" and "s2i". (default "pack")
+  -b, --builder string          builder to use when creating the underlying image. Currently supported builders are "pack" and "s2i". (default "s2i")
```

Comment:
This seems to have picked up a local change unnecessarily

Reply:
Ah yes, #1424 had not yet been merged!  Rebased...